powersimdata/input/change_table.py

```diff
@@ -789,17 +794,92 @@ def _get_df_with_new_elements(self, table):
             'branch', 'bus', 'dcline', or 'plant'.
         :return: (*pandas.DataFrame*) -- the post-transformation table.
         """
```

Comment:
It is kind of confusing that this function is named `_get_df_with_new_elements` now that we use it to not only add new objects but also remove some.